aux/example_config_files/protopipe/dl1_test.yaml

```diff
@@ -0,0 +1,107 @@
+# General informations
+General:
+ config_name: 'lapalma_N_fullarray_MARStailcut_corrected_minNtel2'
+ site: 'north'  # North or South
+ array: 'full_array'  # subarray_LSTs, subarray_MSTs, full_array
+ cam_id_list: ['LSTCam', 'NectarCam'] # Camera identifiers (Should be read in scripts?)
+
+# Cleaning for reconstruction
+ImageCleaning:
+
+ # Cleaning for reconstruction
+ biggest:
+  tail:  #
+   thresholds:  # picture, boundary
+    - LSTCam: [6, 3]
+    - NectarCam: [8, 4]  # TBC
+    - FlashCam: [0,0] 
```

Comment:
won't that accept all pixels? Later, it's set to `[8,4]`, so should at least be consistent.